coderedcms/models/page_models.py

```diff
@@ -133,8 +133,9 @@ class Meta:
 
     # Subclasses can override these fields to enable custom
     # ordering based on specific subpage fields.
-    index_order_by_default = '-first_published_at'
+    index_order_by_default = 'default'
```

Comment:
I would prefer to see this be an empty string. Technically it would not work if there was a field called "default".